apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx

```diff
@@ -7,21 +7,21 @@ import ResourceCard from "../../components/ResourceCard/ResourceCard";
 import { getResources } from "../../getResources";
 import { client } from "@/lib/sanity/client";
 
+import HeadingCard from "../../components/HeadingCard/HeadingCard";
```

Comment:
Chore: rearrange import order to reflect above my comment. I see that styles was left there where it was as an artifact from last year, but can you move that as well to be the last import?